storage/cloudspanner/tree_storage.go

```diff
@@ -391,12 +408,10 @@ func (t *treeTX) getSubtree(ctx context.Context, rev int64, id storage.NodeID) (
 		if st.Prefix == nil && len(stID) == 0 {
 			st.Prefix = []byte{}
 		}
-		// We've got what we want, tell spanner to stop reading by returning
-		// not-really-an-error:
-		return errFinished
+		return nil
 	})
-	if err == errFinished {
-		err = nil
+	if drain > 0 {
+		glog.Warningf("getSubtree drained %d", drain)
```

Comment:
Not sure how you can get this given the `LIMIT 1` per subtree?

Reply:
removed - I was just verifying that.